pkg/deployment/agent.go

```diff
@@ -58,7 +58,12 @@ func (a *Agent) Get() *appsv1.DaemonSet {
 	configRest := util.GetPort("--http-server.host-port=", args, 5778)
 	jgCompactTrft := util.GetPort("--processor.jaeger-compact.server-host-port=", args, 6831)
 	jgBinaryTrft := util.GetPort("--processor.jaeger-binary.server-host-port=", args, 6832)
-	adminPort := util.GetPort("--admin-http-port=", args, 14271)
+	var adminPort int32
```

Comment:
We are reusing the same code in 6 places: it would probably be a good idea to have a func in the `utils` package. Perhaps something like `AdminPortFromArgs(args) int32`? This should also be accompanied from unit tests.

Reply:
Thanks for the feedback! I made the util function alongside the unit tests.